ethereum/test/test_basic_inbound_channel.js

```diff
@@ -0,0 +1,123 @@
+require("chai")
+  .use(require("chai-as-promised"))
+  .should();
+
+const BasicInboundChannel = artifacts.require("BasicInboundChannel");
+const MockApp = artifacts.require("MockApp");
+
+const { ethers } = require("ethers");
+
+const makeCommitment = (messages) => {
```

Comment:
@denalimarsh thought you may be interested in this - A much simpler way to generate commitments on the JS side for unit tests.